sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -418,6 +418,35 @@ class StreamSuite extends StreamTest {
     assert(OutputMode.Update === InternalOutputModes.Update)
   }
 
+  override protected def sparkConf: SparkConf = super.sparkConf
+    .set("spark.redaction.string.regex", "file:/[\\w_]+")
+
+  test("explain - redaction") {
```

Comment:
This is just a SS example. I believe we have more such cases.

Reply:
Is it because you want to use `explainInternal`, so you write a SS test?

Reply:
Just because I found a potential security issue in SS.